interp/constrexpr_ops.mli

```diff
@@ -58,6 +58,11 @@ val mkCLambdaN : Loc.t -> local_binder list -> constr_expr -> constr_expr
 val mkCProdN : Loc.t -> local_binder list -> constr_expr -> constr_expr
 (** Same as [prod_constr_expr], with location *)
 
+val fresh_var_fwd : (Names.Id.t list -> Constrexpr.constr_expr -> Names.Id.t) ref
```

Comment:
This should probably be done using the [`Hook`](https://github.com/coq/coq/blob/trunk/lib/hook.mli) API rather than a `ref`.

An ocamldoc comment explaining what goes in the hook would be welcome too.